src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -1641,6 +1641,7 @@ object SymDenotations {
        */
       def isCachable(tp: Type): Boolean = tp match {
         case _: TypeErasure.ErasedValueType => false
+        case tp: TypeRef if tp.symbol.isClass => true
```

Comment:
down to 22secs from 60secs with this change.
Now, need to do something with `compareAliasedRefined` to get back to 2.5secs.